<launch>
    <arg name="gps_topic" default="/dji_sdk/gps_position" />
    <arg name="attitude_topic" default="/dji_sdk/attitude" />
    <arg name="localPos_topic" default="/dji_sdk/local_position" />
    <arg name="height_takeoff" default="/dji_sdk/height_above_takeoff" />

    <node pkg="riser_inspection" type="local_controller_node" name="riser_inspection" output="screen">
        <param name="gps_topic" type="string" value="$(arg gps_topic)" />
        <param name="attitude_topic" type="string" value="$(arg attitude_topic)" />
        <param name="height_takeoff" type="string" value="$(arg height_takeoff)" />
        <param name="local_position_topic" type="string" value="$(arg localPos_topic)" />
        <param name="root_directory"    type="string"   value="/home/jetson/Documents"/>
        <param name="riser_distance"    type="int"      value="5"/>     <!--METERS-->
        <param name="riser_diameter"    type="int"      value="300"/>   <!--MILLIMETER-->
        <param name="horizontal_points" type="int"      value="5"/>     <!--Nº of horizontal stops-->
        <param name="vertical_points"   type="int"      value="4"/>     <!--Nº of vertical stops at each horizontal point-->
        <param name="delta_H"           type="int"      value="15"/>    <!--DEGREES-->
        <param name="delta_V"           type="int"      value="300"/>   <!--MILLIMETERS-->
    </node>
</launch>
